### ISSUE TITLE
Q about the deployment guide / BW overhead

### ISSUE THREAD

**Issue by oviano** [Haivision/srt#656]
Apologies for the formatting but I have a Q about this table on page 26 of the deployment guide (downloadable from the srtalliance website); why is less bandwidth overhead required for lossier networks - wouldn’t it be the opposite?

3. Using the following table*, find the RTT Multiplier and Bandwidth Overhead values that correspond to your measured Packet Loss Rate:
Worst Case Loss Rate (%)
RTT Multiplier
Bandwidth Overhead (%)
Minimum SRT Latency (for RTT <= 20 ms)
<= 1
3
33
60

<= 3
4
25
80

<= 7
5
20
100

<= 10
6
17
120
* This table takes into account constant loss and burst loss. See “Packet Loss Rate” on page 28 for more information.

**Comment by maxlovic** [comment]
You mean this
![image](https://user-images.githubusercontent.com/12700120/56748201-b06dbc80-677f-11e9-8207-34978fe34694.png)

**Comment by oviano** [comment]
Yes, thank you, that's the one. I don't really understand why the bandwidth overhead requirement decreases as the loss rate increases.

I thought that more losses = more retransmissions = more overhead....but I expect I am misunderstanding something.

**Comment by maxlovic** [comment]
@oviano That is a good question. 😄 So far looks like the formula for the table is `max(1.65 * Packet_loss_rate; 100 / RTT_multiplier)`.
`100 / RTT_multipllier` – not sure about the background of this formula.
The following seems to be the initial motivation:
> Constant loss means that the channel will lose packets at a constant rate.  In this case, the SRT overhead is basically lower bound limited to 1.65 * Packet_loss_rate
Burst loss means the channel will lose multiple consecutive packets up to the the SRT latency buffer size worth of packets.   In this case, the SRT overhead is lower bound limited to 100 / RTT_multiplier.

Will keep you updated.

**Comment by andywangevertz** [comment]
I had this same observation as my bandwidth overhead was less than 1% and my worst case loss rate was 26%. My guess is that the name is kind of misleading. It should read 'max playable/allowance packet loss rate'. As you can read from the table, the RTT multiplier should read 'max resent allowance' as we have less bandwidth overhead which means we could have more resent thus we could endure more packet loss and the result is still acceptable. As a trade off, the more resent the more latency we need.
